DNN Platform/Library/Obsolete/ModuleController.cs

```diff
@@ -104,6 +104,18 @@ public void DeleteTabModuleSettings(int tabModuleId)
         [EditorBrowsable(EditorBrowsableState.Never)]
         [Obsolete("Deprecated in DNN 7.3.  Please use the ModuleSettings property of the ModuleInfo object")]
         public Hashtable GetModuleSettings(int ModuleId)
+        {
+            var cacheKey = string.Format(DataCache.ModuleSettingsByModuleIdCacheKey, ModuleId);
+            var moduleSettings = ModuleId <= 0 ? GetModuleSettingsInternal(ModuleId) :
```

Comment:
Why there is a check for ModuleId <= 0  is there any difference with positive ModuleId?

Reply:
-1 usually equals Null in DNN - e.g. for a newly created module, which hasn't been saved yet.